test/OpenTelemetry.Instrumentation.AspNetCore.Tests/MetricTests.cs

```diff
@@ -497,39 +360,7 @@ private static List<MetricPoint> GetMetricPoints(Metric metric)
         }
     }
 
-    private static void AssertMetricPoints_Old(
-        List<MetricPoint> metricPoints,
-        List<string> expectedRoutes,
-        int expectedTagsCount)
-    {
-        // Assert that one MetricPoint exists for each ExpectedRoute
-        foreach (var expectedRoute in expectedRoutes)
-        {
-            MetricPoint? metricPoint = null;
-
-            foreach (var mp in metricPoints)
-            {
-                foreach (var tag in mp.Tags)
-                {
-                    if (tag.Key == SemanticConventions.AttributeHttpRoute && tag.Value.ToString() == expectedRoute)
-                    {
-                        metricPoint = mp;
-                    }
-                }
-            }
-
-            if (metricPoint.HasValue)
-            {
-                AssertMetricPoint_Old(metricPoint.Value, expectedRoute, expectedTagsCount);
-            }
-            else
-            {
-                Assert.Fail($"A metric for route '{expectedRoute}' was not found");
-            }
-        }
-    }
-
-    private static KeyValuePair<string, object>[] AssertMetricPoint_New(
+    private static KeyValuePair<string, object>[] AssertMetricPoint(
```

Comment:
Not related to this PR. Does this method need to return the attributes? It doesn't get used by the caller.

Reply:
good catch, fixed.